apps/rule-manager/test/db/RuleManagerSpec.scala

```diff
@@ -208,7 +208,9 @@ class RuleManagerSpec extends FixtureAnyFlatSpec with Matchers with AutoRollback
 
       RuleManager.destructivelyPublishRules(allRules, bucketRuleResource)
 
-      DbRuleDraft.findAll() shouldMatchTo allRules
+      DbRuleDraft.findAll() shouldMatchTo allRules.map(rule =>
+        rule.copy(isPublished = !rule.ignore)
```

Comment:
The rules we've created on L204 should be published when `ignore` is false when we call `destructivelyPublishRules`. This is our one-off spreadsheet import operation.

So the fact that `isPublished = false` when `rule.ignore` = false is really a side-effect of that process, based on the one-time import. It's possible in the future that these values could disagree.

We could just drop the `ignore` column. Its value is that it can tell us something historical about an imported rule – but ignored rules are not used, and I think it's somewhat unlikely we will care about them in the future.

Reply:
I see, thanks for the note 👍